IrisSoftware/widgets.py

```diff
@@ -219,7 +219,7 @@ def keyPressEvent(self, event: QtGui.QKeyEvent) -> None:
         return super().keyPressEvent(event)
 
     def __drawCircles(self):
-        gridSize = 5
+        gridSize = 13
```

Comment:
```suggestion
        gridSize = 5
```
Should probably revert this back to 5, unless we want to calibrate 13^2 circles 😄

Reply:
I think it would be okay to ask the user to spend a few minutes calibrating, especially if the volume of extra data is what allows a correct prediction to be made. Here's an example of an expensive commercial software product with a really long calibration phase: https://app.gazerecorder.com/.

But until Nalin can get gaze tracking working more accurately, I guess we don't need to keep it as high as 13x13 in our main branch. We can always revisit later to figure out the minimum amount of calibration data necessary to get a model working accurately.